cli/injector/main.go

```diff
@@ -291,21 +285,79 @@ func injectAndWait(cmd *cobra.Command, args []string) {
 		if err := inj.Inject(); err != nil {
 			errOnInject = true
 
-			handleMetricError(ms.MetricInjected(false, cmd.Name(), nil))
+			if sendToMetrics {
+				handleMetricError(ms.MetricInjected(false, kind, nil))
+			}
+
 			log.Errorw("disruption injection failed", "error", err)
 		} else {
-			handleMetricError(ms.MetricInjected(true, cmd.Name(), nil))
-			log.Info("disruption injected, now waiting for an exit signal")
+			if sendToMetrics {
+				handleMetricError(ms.MetricInjected(true, kind, nil))
+			}
+
+			log.Infof("disruption %s injected", inj.GetDisruptionKind())
+		}
+	}
+
+	if errOnInject {
+		log.Errorf("an injector could not inject the disruption successfully, please look at the logs above for more details")
+	}
+
+	return errOnInject
+}
+
+// clean clean all the disruptions using the list of injectors
+// returns true if cleanup succeeded, false otherwise
```

Comment:
This comment is backwards, we return true if cleanup had an error, false otherwise. Would it be better to return an error here (or a multi error), rather than a bool?

Reply:
I fixed the function return value to match the comment. The inject / clean functions were previously returning the list of errors but it was unused in the code after as we're already logging every error inside of the functions.